gf-webapp-admin/src/utils/enumToArray.tsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/**
+ * @param anyEnum - any enum
+ * @returns
```

Comment:
add description

Reply:
done

Reply:
try to add "@descriptoin" on the line before param and describe "@return"

Reply:
done